js/templates/userPage/userPageHome.html

```diff
@@ -0,0 +1,81 @@
+<div class="flexRow gutterH">
+  <div class="col4 gutterVMd">
+    <div class="js-userShort">
+      <!-- the userShort template is injected here-->
+    </div>
+    <% if(ob.isModerator) { %>
```

Comment:
If the model has a moderator flag, why do we need isModerator. Is there some business logic to determine that on top of what is stored in the server?